src/note.cpp

```diff
@@ -266,12 +266,14 @@ const TabGrp *Note::IsTabGrpNote() const
     return vrv_cast<const TabGrp *>(this->GetFirstAncestor(TABGRP, MAX_TABGRP_DEPTH));
 }
 
-std::u32string Note::GetTabFretString(data_NOTATIONTYPE notationType) const
+std::u32string Note::GetTabFretString(data_NOTATIONTYPE notationType, bool &overline) const
```

Comment:
What is the advantage to hand `overline` over from here instead of getting it in `ViewTab` directly?

Reply:
There are several different notations for the 6th, 7th ... courses that I have not yet implemented.  Some of these use overlines.  As the overline is part of the character it seemed better to put the logic in note.cpp where the characters are selected, rather than in view_tab.cpp where they are drawn.    The implementation of these other notations requires some means in MEI of specifying which to use.